### ISSUE TITLE
3dsx on 2DS/o3DS?

### ISSUE THREAD

**Issue by DarthJahus** [masterfeizz/DeSmuME#8]
Is the latest release `.3dsx` working on 2DS?

Moreover, could you please add some instructions on the readme.md file?

**Comment by masterfeizz** [comment]
No, the .3dsx only works on N3DS with svchax support. The instructions are on the release page, just rename the rom to game.nds and place in the root of sdcard.

Btw, the emulator is way to slow on the 2DS to be useful
